model_analyzer/analyzer.py

```diff
@@ -134,6 +134,11 @@ def profile(self, client: TritonClient, gpus: List[GPUDevice], mode: str,
                         self._get_report_command_help_string(
                             model.model_name()))
 
+            if self._metrics_manager.encountered_perf_analyzer_error():
+                logger.warning(f"Perf Analyzer encountered an error when profiling one or more configurations. " \
+                      f"See {PA_ERROR_LOG_FILENAME} for further details.\n")
```

Comment:
It is, but I don't think it's a good idea. They can be quite long and I think it's too much info for the summary output and might confuse the user about what's going on.

Reply:
In the future it would be good to parse/bucket the failures into common cases and communicate it to stdout so the users don't have to read and parse a huge log

Reply:
and, the output should point to the full file, not just the file name. `{self._config.export_path}/{PA_ERROR_LOG_FILENAME}` (or perf_output_path if you update it to be there instead)

Reply:
Hopefully PA errors are rare enough that this doesn't matter!
I've pointed the output to full file path.